lib/screens/fake_home_page.dart

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Do we still need this widget?

Reply:
No, it is not needed. This widget is just for my own checking whether the user authentication is done.
I will delete this in the later commit.